group_vars/pulsarservers.yml

```diff
@@ -0,0 +1,54 @@
+
+galaxy_server_hostname: "galaxy.atgm.avans.nl" # Important!!!
+# Put your Galaxy server's fully qualified domain name (FQDN) (or the FQDN of the RabbitMQ server) above.
+
+pulsar_root: /mnt/pulsar
+
+pulsar_pip_install: true
+pulsar_pycurl_ssl_library: openssl
+pulsar_systemd: true
+pulsar_systemd_runner: webless
+
+pulsar_create_user: true
+pulsar_user: {name: pulsar, shell: /bin/bash}
+
+pulsar_optional_dependencies:
+  - pyOpenSSL
+  # For remote transfers initiated on the Pulsar end rather than the Galaxy end
+  - pycurl
+  # drmaa required if connecting to an external DRM using it.
+  - drmaa
+  # kombu needed if using a message queue
+  - kombu
+  # amqp 5.0.3 changes behaviour in an unexpected way, pin for now.
+  - 'amqp==5.0.2'
+  # psutil and pylockfile are optional dependencies but can make Pulsar
+  # more robust in small ways.
+  - psutil
+
+pulsar_yaml_config:
+  staging_directory: "{{ pulsar_staging_dir }}"
+  persistence_directory: "{{ pulsar_persistence_dir }}"
+  tool_dependency_dir: "{{ pulsar_dependencies_dir }}"

```

Comment:
I think these are set relative to `pulsar_root`, we should consider if there is enough space here or if we should set this to one of the /mnt drives or so. (Maybe the same user drive where we create homedirs?)

Reply:
i checked the directories. Since the data is temporary we should definitly have enough storage in the mnt/student_files ((project_directory)). Is there a way to perhaps reserve storage space. ? so for example set this directory to always have 500 gb reserved for the galaxy jobs?

Reply:
fair @ temporary.  We could partition it in the future when we rebuild those machines from scratch.

Reply:
then we keep the partition on our to do list once we will wipe the other computers :) for now plenty of room in the student mnt